packages/debug/package.json

```diff
@@ -12,17 +12,16 @@
     "@theia/process": "^0.3.14",
```

Comment:
Should changes to `theia/debug` be put in a separate PR?

Reply:
I separated in 3 different commits:
- added a named logger for easier debugging of the debug extension when you make contributions
- used the preferred async/await notation while I was at it
- added the cpp debugger

I didn't think the changes I made to the debug extension were worth a dedicated PR, so I made the two extra commits here. If you really want it out I could do it, me I am fine like this :)

Reply:
I don't mind